lws_config.h.in

```diff
@@ -61,3 +61,5 @@
 
 /* use SHA1() not internal libwebsockets_SHA1 */
 #cmakedefine LWS_SHA1_USE_OPENSSL_NAME
+
+${LWS_SIZEOFPTR_CODE}
```

Comment:
Hrm, I mean how did you get this to work at all? Because this code is broken.... You'll have a header with a random `8` or whatever in it, no define or nothing...